tabs-tabs-tabs/tabs.js

```diff
@@ -12,6 +12,30 @@ function firstUnpinnedTab(tabs) {
   }
 }
 
+function listTabs() {
+  chrome.tabs.query({
+    currentWindow: true
+  }, function(tabs) {
+    var tabsList = document.getElementById('tabs-list'),
+        currentTabs = document.createDocumentFragment();
+
+    tabsList.innerHTML = '';
+
+    for (var tab of tabs) {
+      if (!tab.active) {
```

Comment:
This confused me, because the number of listed tabs is always less than the number of tabs I have. Also because the popup persists as the tab switches this gets out of sync. I'm not sure what a nice solution is there though. Perhaps is we limit it to 5 or so (see other comments) this won't be too bad.